apis/taskcluster.py

```diff
@@ -7,17 +7,17 @@
 import json
 import jsone
 import requests
+from urllib.parse import quote_plus
 
 from components.utilities import Struct
 from components.logging import logEntryExit, logEntryExitNoArgs, LogLevel
 from components.providerbase import BaseProvider, INeedsCommandProvider, INeedsLoggingProvider
 
-RETRIGGER_NUMBER = 3
+RETRIGGER_TOTAL = 4
```

Comment:
Can we add a comment here about why we picked 4 so that we don't unintentionally bump it higher in the future?